code/datums/elements/clickbox.dm

```diff
@@ -0,0 +1,99 @@
+#define CLICKBOX_LAYER FLOAT_LAYER-999
```

Comment:
This seems hacky, could you put an actual layer here instead of doing `FLOAT_LAYER-999`? It seems you want to get the lowest possible layer, so this could be moved into its own plane instead.

Reply:
Please, don't add more planes. This is exactly what float layer is for. A plane would not work with this at all

Reply:
I thought FLOAT_LAYER - X moves it to a layer below the parent by an amount of X, or does it not work the same way as FLOAT_PLANE?